components/TextFields/src/ContainedInputView/MDCBaseTextField.h

```diff
@@ -105,4 +117,18 @@
  */
 - (nonnull UIColor *)textColorForState:(MDCTextControlState)state;
 
+/**
+ Sets the assistive label color for a given state.
+ @param assistiveLabelColor The UIColor for the given state.
+ @param state The MDCTextControlState.
+ */
+- (void)setAssistiveLabelColor:(nonnull UIColor *)assistiveLabelColor
+                      forState:(MDCTextControlState)state;
+
+/**
+ Returns the assistive label color for a given state.
+ @param state The MDCTextControlState.
+ */
+- (nonnull UIColor *)assistiveLabelColorForState:(MDCTextControlState)state;
```

Comment:
In the future if clients want different values for the leading and trailing the value gotten from this getter will not be obvious.

Reply:
It sort of sounds like you want separate APIs for the leading and trailing assistive label colors

Reply:
yes please I think it will be more scale able if clients want to add them.

Reply:
Done.